File: content/en/api/v1/examples/UpdateDowntime.java
import com.datadog.api.v2.client.api.DowntimesApi;

public class DowntimesApiExample {

    public static void main(String[] args) {
        DowntimesApi apiInstance = new DowntimesApi();
        Long downtimeId = 123456; // Long | ID of the downtime to update
        Downtime body = ; // Downtime | 
        try {
            Downtime result = apiInstance.updateDowntime(downtimeId, body);
            System.out.println(result);
        } catch (ApiException e) {
            System.err.println("Exception when calling DowntimesApi#updateDowntime");
            e.printStackTrace();
        }
    }
}

File: content/en/api/v1/examples/UpdateDowntime.go
package example

import (
	"context"
	"os"

	. "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
)

ctx := context.WithValue(
	context.Background(),
	ContextAPIKeys,
	map[string]APIKey{
		"apiKeyAuth": {
			Key: os.Getenv("DD_CLIENT_API_KEY"),
		},
		"appKeyAuth": {
			Key: os.Getenv("DD_CLIENT_APP_KEY"),
		},
	},
)
config := NewConfiguration()
api := NewAPIClient(config)

model, httpresp, err := api.DowntimesApi.updateDowntime(ctx).Execute()
if err != nil {
	log.Fatalf("API Error %s: %v", err.(GenericOpenAPIError).Body(), err)
}
log.Printf("Response %v", model)
